app/components/recipeCard.jsx

```diff
@@ -31,9 +31,16 @@ export default function RecipeCard({ N, recipeName, imageUrl, recipeId }) {
     setShowAlert(true);
   };
 
-  const handleConfirm = () => {
+  const handleConfirm = async () => {
     setShowAlert(false);
-    console.log("Confirmed 'Hacer Receta'");
+    try {
+      const response = await registerRecipeConsumption(recipeId);
+      console.log(response);
+      alert(`${response.message}`);
```

Comment:
Is this alert showing text that the user will understand, or is it mainly for our debugging purposes?